### PR TITLE
Issue18

### PR DESCRIPTION
**Pull request by KHafner**
# Pull Request Template

## Checklist:

- [x] I have written test cases for this change, and my code passed the test
- [x] I have updated documentation to reflect this change
- [x] I have attached this PR to an issue/user story

### CODE REVIEW

**Review by jules203 — APPROVED** [review]
Looks good!

File: service/routes.py [hunk 1]
```diff
@@ -361,6 +361,40 @@ def delete_shopcarts(shopcart_id):
...
+            logger.info("Shopcart with ID [%s] is empty.", shopcart_id)
+            return not_found("Not found")
+        shopcart_items_list = [item.serialize() for item in shopcart_items]
+
+        #once we have the list of shopcart items we can send in JSON format to the orders team
+        #SEND shocart_items_list TO ORDERS TEAM
+
+        #empty the shopcart
+        for item in shopcart_items:
+            item.delete()
```
Comment (iLtc) [inline service/routes.py:388]: The `shopcart.delete()` below already did this for you.

**Review by iLtc — APPROVED** [review]
LGTM, thanks!

File: service/routes.py (hunk 1 above)
Comment (KHafner) [reply service/routes.py]: thanks, I updated it

**Review by gauridhawan — APPROVED** [review]

**Pull request merged**